scm/GitManager.go

```diff
@@ -57,25 +167,19 @@ func (a GitManager) Commit(dir bugs.Directory, commitMsg string) error {
 
 	}
 
-	var deletedIdentifiers []string
-	if a.Autoclose == true {
-		deletedIdentifiers = a.getDeletedIdentifiers(dir)
-	} else {
-		deletedIdentifiers = []string{}
-	}
-	if len(deletedIdentifiers) > 0 {
-		commitMsg = fmt.Sprintf("%s\n\nCloses %s\n", commitMsg, strings.Join(a.getDeletedIdentifiers(dir), ", closes "))
-	} else {
-		commitMsg = fmt.Sprintf("%s\n", commitMsg)
+	msg := a.commitMsg(dir)
+	if len(msg) == 0 {
```

Comment:
I think Travis is failing because of this change. The tests are expecting the parameter to commit to be a subject for the commit message (the first line), (which means we can easily identify the commit in short form git logs) not a backup if there's no other commit message.

See GitManager_test:203